beets/util/queue.py

```diff
@@ -146,10 +153,9 @@ def put(self, item):
         return immediately.
         """
         with self.not_full:
-            if not self._invalidated and self._maxsize > 0:
-                while (not self._invalidated and
-                       len(self._queue) >= self._maxsize):
-                    self.not_full.wait()
+            while (not self._invalidated and self._is_full()):
```

Comment:
Parentheses are unnecessary here (since it fits on one line).

Reply:
Right, thanks for spotting this!